app/cli/Main.hs

```diff
@@ -128,6 +135,11 @@ exec args
     | args `isPresent` command "address" && args `isPresent` command "list" = do
         wid <- args `parseArg` longOption "wallet-id"
         print (wid :: WalletId)
+        manager' <- newManager defaultManagerSettings
+        res <- runClientM (listAddresses (ApiT wid) (Just $ ApiT Unused)) (mkClientEnv manager' (BaseUrl Http "localhost" 8080 ""))
+        case res of
+            Left err -> putStrLn $ "Error: " ++ show err
+            Right address -> print address
```

Comment:
Please refer to https://github.com/input-output-hk/cardano-wallet/issues/96 about the scope of the CLI commands for now :s ... This endpoint isn't in scope, nor implemented in the API at the moment.